mesonbuild/modules/i18n.py

```diff
@@ -62,6 +62,10 @@ def merge_file(self, state, args, kwargs):
 
         kwargs['command'] = ['msgfmt', '--' + file_type,
                              '--template', '@INPUT@', '-d', podir, '-o', '@OUTPUT@']
+
+        extra_args = mesonlib.stringlistify(kwargs.get('args', []))
+        kwargs['command'].append(extra_args) if extra_args
```

Comment:
Oops, first time in Python, I guess I copied wrong (well, there’s `else None` in the part I copied but here it seemed wrong)